src/components/AdminUi.tsx

```diff
@@ -57,6 +57,7 @@ const AdminUI: React.FC<AdminUIProps> = ({
                         min={0}
                         max={totalPages}
                         step={1}
+                        overRideValue={page}
```

Comment:
Nit: "override" is one word
```suggestion
                        overrideValue={page}
```